main.py

```diff
@@ -2,20 +2,76 @@
 import git
 import glob
 import json
+import re
+import sys
 import os
 import shutil
 import subprocess
 
+if os.path.isdir('./cobalt/tools'):
+    sys.path.append(os.path.abspath('./cobalt/tools'))
+    import convert_dep_to_squashed
+else:
+    raise ModuleNotFoundError('third_party/depot_tools is not found')
+
+# Not perfect because it'll match a mismatched quotes
+_DEPS_FIELD_REGEX = re.compile(r'(\s*)[\'\"](src\/)?(\w+)[\'\"]\s*:')
+SUBTREE_SCRATCH_DIR = '/tmp/cobalt/subtree_scratch'  # Temporary directory to stage repo clones
```

Comment:
Do a `os.path.join(tempfile.gettempdir(),"cobalt","subtree_scratch")` please just to be polite. I know this will work 100% of the cases but .. we never know when we want to turn this code into a bigger tool.